js/coinmate.js

```diff
@@ -281,6 +473,10 @@ module.exports = class coinmate extends Exchange {
         return await this.privatePostCancelOrder ({ 'orderId': id });
     }
 
+    nonce () {
```

Comment:
Is it required? They won't accept more than 100 requests per minute, that could be roughly compared to 1 per minute... What do you think?

Reply:
It allows for faster bursts or requests, and in the case where you are hitting the exchange in multiple threads (without in-build throttling) it reduces the chance of nonce-reuse.  In general I would always favour the highest resolution nonce that the exchange supports.